core/frontend/src/components/parameter-editor/InlineParameterEditor.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <v-form
+    ref="form"
+    v-model="is_form_valid"
+    @submit.prevent="saveEditedParam()"
+  >
+    <template v-if="!custom_input && param?.bitmask">
+      <v-checkbox
+        v-for="(key, keyvalue) in param?.bitmask"
+        :key="keyvalue"
+        v-model="selected_bitflags"
+        dense
+        hide-details
+        :loading="waiting_for_param_update"
+        :label="key"
+        :value="2 ** keyvalue"
+      />
+    </template>
+    <v-autocomplete
+      v-else-if="!custom_input && Object.entries(param?.options ?? []).length > 10"
```

Comment:
is there an reason for why enabling it if bigger than 10 ? Why not always ?

Reply:
well, that's your code :eyes: , I just copied it over, but the logic is that we don't need autocomplete if we have few choices